tests/unit/test_config.yaml

```diff
@@ -0,0 +1,15 @@
+namespaces:
```

Comment:
Are this file and the one below it used? It does not look like it. And they use relative paths on your machine which won't work for everyone.